filament/src/PostProcessManager.cpp

```diff
@@ -235,22 +204,29 @@ FrameGraphId<FrameGraphTexture> PostProcessManager::fxaa(FrameGraph& fg,
                 });
                 data.rt = builder.createRenderTarget(data.output);
             },
-            [=](FrameGraphPassResources const& resources,
+            [=, engine = &engine](FrameGraphPassResources const& resources,
```

Comment:
couldn't you just write `[=, &engine]` here?

Reply:
Yes, good catch